src/settings/toml/manifest.rs

```diff
@@ -487,6 +487,10 @@ impl Manifest {
             StdOut::help(
                 &format!("You will need to update the following fields in the created {} file before continuing:", toml_msg)
             );
+            StdOut::help(&format!(
+                "You can find your {} in the right sidebar of your account's Workers page, and {} in the right sidebar of a zone's overview tab at {} (if you have only a workers.dev domain, you can skip adding the {} )",
+                account_id_msg, zone_id_msg, dash_url, zone_id_msg
```

Comment:
You're using a variable that was removed sometime in the last 6 months. You could see how it was calculated before and do the same here.

Reply:
Makes sense....

Reply:
added variable, a maintainer needs to approve the workflow...

Reply:
https://github.com/cloudflare/wrangler/pull/1596/checks?check_run_id=3194423646 Do you have any idea as to why this is failing, there isn't much context.....

Reply:
I think it was the additional new line.... and I have removed it now.....